WordPress/Classes/ViewRelated/Reader/ReaderDetailCoordinator.swift

```diff
@@ -124,6 +145,26 @@ class ReaderDetailCoordinator {
         })
     }
 
+    /// Requests a ReaderPost from the service and updates the View.
+    ///
+    /// Use this method to fetch a ReaderPost from a URL.
+    /// - Parameter url: a post URL
+    private func fetch(_ url: URL) {
+        service.fetchPost(at: postURL,
+                          success: { [weak self] post in
+                            guard let post = post else {
+                                return
+                            }
+
+                            self?.post = post
+                            self?.renderPostAndBumpStats()
+                            self?.view?.show(title: post.postTitle)
+        }, failure: { [weak self] error in
+            DDLogError("Error fetching post for detail: \(String(describing: error?.localizedDescription))")
+            self?.view?.showError()
```

Comment:
Shouldn't we pass the error to showError ?

Reply:
This is out of the scope for this task.

But even in the subsequent, we don't make distinction on what error happened. Any error will always display the same message, same for the current reader (you can see [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/Reader/ReaderDetailViewController.swift#L419)).